salt/modules/win_lgpo.py

```diff
@@ -2672,9 +2672,12 @@ def __virtual__():
     '''
     Only works on Windows systems
     '''
-    if salt.utils.platform.is_windows() and HAS_WINDOWS_MODULES:
-        return __virtualname__
-    return False
+    if not salt.utils.platform.is_windows():
+        return False, 'win_lgpo: Not a Windows System'
+    if not HAS_WINDOWS_MODULES:
+        return False, 'win_lgpo: Required modules failed to load'
+    log.debug('win_lgpo: LGPO module loaded successfully')
```

Comment:
Can you remove this debug statement? This will flood the logs on a windows system.